flex/validation/common.py

```diff
@@ -323,13 +324,46 @@ def validate_allof_anyof(value, sub_schemas, context, method, **kwargs):
 
 
 def generate_allof_validator(allOf, context, **kwargs):
-    return functools.partial(validate_allof_anyof, sub_schemas=allOf, context=context, method=all)
+    if 'resolved_refs' not in context:
+        context['resolved_refs'] = []
```

Comment:
*nitpick*

Generally using `context.setdefault('resolved_refs', [])` is a bit cleaner and easier to read.